### ISSUE TITLE
Show the version number

### ISSUE THREAD

**Issue by sindresorhus** [SwiftDocOrg/swift-doc#120]
The docs site should show the current version (Git tag) of the project.

Before:

<img width="666" alt="Screenshot 2020-05-21 at 15 09 19" src="https://user-images.githubusercontent.com/170270/82533165-53585c00-9b75-11ea-80ff-efe5a0c3094f.png">

After:

<img width="695" alt="Screenshot 2020-05-21 at 15 11 01" src="https://user-images.githubusercontent.com/170270/82533176-59e6d380-9b75-11ea-83fc-d7e0f7376356.png">

**Comment by mattt** [comment]
I agree that we should show version information, though I'm not 100% sold on that particular design (at least so long as we have a <sup>BETA</sup> there)
